MAINTAINERS.yml

```diff
@@ -2357,6 +2365,15 @@ West:
   labels:
     - manifest-hal_ti
 
+"West project: hal_wurthelektronik":
+  status: odd fixes
+  collaborators:
+    - mah-eiSmart
+  files:
+    - modules/Kconfig.wurthelektronik
+  labels:
+    - manifest-hal_wurthelektronik
```

Comment:
why is this "odd fixes"? based on https://github.com/zephyrproject-rtos/zephyr/issues/47021, @mah-eiSmart should be maintaining it?

Reply:
Sorry, copy paste, let me change it. Thanks for catching it.